src/genSchema/generateTableSchema.ts

```diff
@@ -96,7 +96,8 @@ export ${outputFields};
 
 import { z } from "zod";
 
-import { ${tableName}InputSchemaGen, ${tableName}OutputSchemaGen } from "../../_generated/${tableName}/${tableName}SchemaGen.js";
+import { ${tableName}InputSchemaGen, ${tableName}OutputSchemaGen } from "../../_generated";
```

Comment:
It must include at least `index.js`, because of ESM

Reply:
Fixed this